tests/system/action/test_action_command_format.py

```diff
@@ -0,0 +1,291 @@
+from unittest.mock import Mock
+
+from openslides_backend.action.action_handler import ActionHandler
+from openslides_backend.action.util.typing import Payload
+
+from .base import BaseActionTestCase
+
+# TODO: parse_action Tests als UnitTests zum Action Handler, die anderen, die das schreib format testen mal schauen
```

Comment:
Please either resolve this or remove it and open an issue for this.

Reply:
I think we should leave it as system test, because it needs to call at minimum the DatastoreService

Reply:
I meant the `TODO` ;) the test itself is fine.